apps/ledger-live-mobile/src/families/evm/StakingDrawer/EvmStakingDrawerProviderIcon.tsx

```diff
@@ -8,6 +8,12 @@ import { KelpDAO } from "~/icons/KelpDAO";
 import { P2P } from "~/icons/P2P";
 import { RocketPool } from "~/icons/RocketPool";
 import type { ColorValue } from "react-native";
+import Coinbase from "~/icons/Coinbase";
```

Comment:
nitpick: could be good to have them sorted alphabatically, also would it make sense to store those stake icons in their own folders? or are some maybe used in other context?